src/styles/block/card.scss

```diff
@@ -0,0 +1,98 @@
+@import "../utils/variables";
+
+.card {
+  margin-bottom: 46px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $card-container-accent-color;
+  margin-right: $card-mar-r;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__description {
+    padding: 40px 16px 16px;
+  }
+
+  &__name {
+    margin: 0%;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 150%;
+    padding-bottom: 4px;
+  }
+
+  &__product-code {
+    padding-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    display: flex;
+    font-size: 10px;
+    line-height: 16px;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    color: $secondary-color;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    padding-bottom: 16px;
+  }
+
+  &__price-current {
+    color: $card-container-accent-color;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  &__buy-button {
+    height: 40px;
+    box-sizing: content-box;
+    line-height: 16px;
+    font-weight: 700;
+    font-size: 14px;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+    border-radius: 5px;
+    background-color: $card-blue-color;
+    text-decoration: none;
+    color: $card-white-color;
+
+    &:hover {
+      background-color: $card-white-color;
+      color: $card-blue-color;
+      border: 1px solid $card-blue-color;
```

Comment:
Add `border: 1px solid transparent` to prevent UI jumping. You should have a border if you add it to the hover state.